Portfolio/index.html

```diff
@@ -33,12 +39,48 @@ <h2>Skills</h2>
         <li>React</li>
         <li>Node js</li>
     </ol>

```

Comment:
The classes 'so' and 'fa' in the <i> tags should be separated by a space to apply both classes correctly.